valkey.conf

```diff
@@ -668,6 +668,22 @@ repl-diskless-sync-max-replicas 0
 #                 during replication.
 repl-diskless-load disabled
 
+# This dual channel replication sync feature optimizes the full synchronization process 
+# between a primary its replicas. When enabled, it reduces both memory and CPU load 
+# on the primary server.
+#
+# How it works:
```

Comment:
At some point you should mention the tradeoff is that the replication stream is stored on the replica. That is basically the tradeoff.

Reply:
That seems OK to me.